gear-programs/vara-tokenizer/app/src/admin_service.rs

```diff
@@ -0,0 +1,40 @@
+use sails_rs::{
+    gstd::msg,
+    prelude::{collections::HashSet, *},
+};
+
+#[derive(Debug)]
+pub(crate) struct AdminConfig {
+    pub admins: HashSet<ActorId>,
+}
+
+static_storage!(AdminConfig);
```

Comment:
It is better to keep it in the program's state from my point of view

Reply:
Methods `storage` and `storage_mut` are generated by macro, which may conflict with other storages.
It is better to use in the service module.